examples/atomic-registry/Makefile

```diff
@@ -21,7 +21,7 @@ install-atomic:
 install-systemd:
 	atomic install $(TEST_IMAGE) $(INSTALLHOST)
 	systemctl enable --now atomic-registry-master.service
-	sleep 10
+	sleep 30
```

Comment:
In a perfect world, this could be replaced with a polling loop -- although, I thought that `systemctl enable` would block until the service was running?